support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -96,12 +108,20 @@ function withProps(props: PropTypes) {
     props.onThirdPartyPaymentAuthorised(paymentAuthorisation);
   };
 
-  const landingPageCopy = props.isReturningContributor && props.landingPageCopyReturningSinglesTestVariant === 'returningSingle' ?
-    returningSingleHeaderCopyAndContributeCopy :
-    defaultHeaderCopyAndContributeCopy;
+  const landingPageCopy = (): CountryMetaData => {
+    if (props.isReturningContributor && props.landingPageCopyReturningSinglesTestVariant === 'returningSingle') {
```

Comment:
should this be below the `US` check?

Reply:
the problem with putting it below the US one is that then I would have to modify the test to exclude the US for a period of time as well

Reply:
per amanda, going to end this test